test_coverage/api_testing.py

```diff
@@ -0,0 +1,188 @@
+from cgi import test
+import json
+import sys
+from fastapi.testclient import TestClient
+from app.api import API
+from data_generators.user_generators import Mentee, Mentor
+from pprint import pprint
+
+
+class TestingPlatform():
+    sys.tracebacklimit = 0
+    client = TestClient(API)
+
+    # Don't change this flag unless you know what you're doing.
+    populate = False
+
+    def test_version(self):
+        """ Tests the current version of the API. """
+
+        response = self.client.get("/version")
+
+        assert response.status_code == 200, \
+            pprint("test_version: Expected Code 200. Got " +
+                   str(response.status_code))
+
+        assert response.json() == {'result': '0.43.8'}, \
```

Comment:
Good job but I am getting this error "test_version: Expected 0.43.8 Got {'result': '0.43.7'}" when I perform the test. May want to change this version or update the version in app/api.py to '0.43.8'. Thanks

Reply:
Once main is merged this problem will be fixed